app/page.tsx

```diff
@@ -62,6 +117,7 @@ export default function HomePage() {
   const router = useRouter();
   const showClientUI = useShowClientUI();
   const [isPopoverOpen, setPopoverOpen] = useState(false);
+  const [isStatePopoverOpen, setStatePopoverOpen] = useState(false);
```

Comment:
```suggestion
  const [toggleStatusPopover, setToggleStatusPopover] = useState(false);
```

Reply:
I think in this case we can call it "is" since it's a boolean value. "toggle" sounds like an action.